services/webnn/ort/graph_builder_ort.cc

```diff
@@ -1416,6 +1479,244 @@ void GraphBuilderOrt::AddGemmOperation(const mojom::Gemm& gemm) {
                         std::move(attributes));
 }
 
+// `GruType` must be `mojom::Gru` or `mojom::GruCell`.
+template <typename GruType>
+  requires(std::is_same_v<GruType, mojom::Gru> ||
+           std::is_same_v<GruType, mojom::GruCell>)
+[[nodiscard]] base::expected<void, mojom::ErrorPtr>
+GraphBuilderOrt::AddGruOperation(const GruType& gru) {
+  const std::string node_name = GenerateNextOperationName(gru.label);
```

Comment:
Follow the naming convention in https://github.com/shiyi9801/chromium/pull/185

Reply:
Done.